Hometask8/src/test/java/homework21/SelenideTest.java

```diff
@@ -4,68 +4,48 @@
 import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
-import static com.codeborne.selenide.Condition.visible;
 import static com.codeborne.selenide.Selenide.*;
 import static org.testng.Assert.assertEquals;
 
 public class SelenideTest {
-    final String userName = "standard_user";
-    final String password = "secret_sauce";
-    final String inventoryPageTitle = "Products";
-    final String title = ".title";
-    final String cartBtnSelector = ".shopping_cart_link";
-    final String cartTitle = "Your Cart";
-    final String checkoutBtnId = "#checkout";
-    final String checkoutTitle = "Checkout: Your Information";
-    final int selectedItem = 2;
-    final String[] checkoutData = {"Kate", "Smith", "98765"};
-    final String overviewTitle = "Checkout: Overview";
     final String productsInCartSelector = ".inventory_item_price";
     final String summarySelector = ".summary_subtotal_label";
-    final String finishBtnId = "#finish";
     final String completeTextSelector = ".complete-header";
     final String completeText = "Thank you for your order!";
+
     @BeforeTest
     public void setup() {
         Configuration.baseUrl = "https://www.saucedemo.com/";
         Configuration.browserSize = "1440x1100";
     }
 
-    public SelenideElement getElement(String selector){
+    public SelenideElement getElement(String selector) {
         return $(selector);
```

Comment:
should be moved to proper Page object class, elements should not be fount in test methods